extensions/VisualEditor/modules/ve-mw/init/ve.init.mw.Target.js

```diff
@@ -1014,6 +1014,10 @@ ve.init.mw.Target.prototype.load = function ( additionalModules ) {
 		data.oldid = this.revid;
 	}
 
+	if ( uri.query.preload ) {
```

Comment:
Probably you want to add something like: ```mw.config.get('wgArticleId') === 0``` (btw. this was already mentioned in the PR)